docs/articles/vs-test-adapter/Adapter-Installation.md

```diff
@@ -4,43 +4,63 @@ uid: vstestadapterinstallation
 
 # Adapter Installation
 
-There are two ways of installing the adapter within Visual Studio. See below for info on how to choose.
-
-* Add it as a NuGet package to your solution. (Requires VS 2012 Update 1 or later)
-* Use the Extension Manager  (Deprecated and pending future removal; see notes below.)
+The adapter is to be installed as a nuget package to each of your test projects.
 
 ## Installing the NuGet Package
 
-To add it is a NuGet package, you must have an active solution, then follow these steps:
+You will most likely want to add a new NUnit Test Project to your solution.  The easiest way is to use the ```dotnet``` command on the command line.
 
-1. From Tools menu, use Library Package Manager and select Manage NuGet packages for solution
-2. In the left panel, select Online
-3. Locate (search for) NUnit 3.0 Test Adapter in the center panel and highlight it
-4. Click 'Install'
-5. In the "Select Projects" dialog, you need to select at least one project to add the adapter to, see notes below.
+Go to your project root and where you want to add your project.
+
+Create a folder for the project with a name matching what your new test project should be named.
+
+Then run the command ````dotnet new nunit```
+
+You will now get a complete nunit test project with the same name as the folder.  You will also have a template unit test class there as a starter.
 
-## Installing With the Extension Manager
+If you have a Visual Studio solution file in the root folder, you can go there and add the new nunit csproj easily:
 
-> [!WARNING]
-> The use of the VSIX extension manager is deprecated.  It still works, but at some time support for it will be removed from Visual Studio, and also newer versions of the adapter. We recommend you try to move to nuget based adapters.
+Assume you are at a solution root, and you either have a solution file, or have just been adding it with ```dotnet new sln```
 
-To install the NUnit Test Adapter using the Extension Manager, follow these steps:
+```cmd
+md Whatever.Test
+cd Whatever.Test
+dotnet new nunit
+cd ..
+dotnet sln add Whatever.Test\Whatever.Test.csproj
+```
 
-1. From within Visual Studio, select Tools | Extension Manager.
-1. In the left panel of the Extension Manager, select Online Extensions
-1. Locate (search for) the NUnit 3.0 Test Adapter in the center panel and highlight it.
-1. Click 'Download' and follow the instructions.
+And you're ready to go !
 
-Use the Extension Manager to ensure that the NUnit 3.0 Test Adapter is enabled.
+### Manually adding it as a package reference to your test projects
 
-## How to choose between Extension and NuGet package
+The recommended set of packages you should install is the framework, the adapter and the analyzer.  The adapter and the framework is required, the analyzer will help you, but is not required to make your project work.  The coverlet.collector is the recommended code coverage package you should go for.
```

Comment:
❓ Why is the coverlet package the recommended package?

Reply:
It is integrated into the Visual Studio Test platform as a collector, from VS version 16.5.